### PR TITLE
Support specifying the NDK and the Oculus dirs in gradle.properties;

### PR DESCRIPTION
**Pull request by liaxim**
both the global and the project one should work.

GearVRf-DCO-1.0-Signed-off-by: Mihail Marinov <m.marinov@samsung.com>

### CODE REVIEW

**Comment by liaxim** [comment]
Just modify either ~/.gradle/gradle.properties or the gradle.properties under the project directory to include these:

```
OVR_MOBILE_SDK=</path/to/oculus>
ANDROID_NDK_HOME=</path/to/ndk>
```

Verified by unsetting all the variables we suggest in the documentation and removing the ndk from the PATH.

File: GVRf/Framework/framework/build.gradle [hunk 1]
```diff
@@ -32,22 +32,41 @@ android {
...
         }
+
+        commandLine ndkbuild, '-C', file('src/main').absolutePath, '-j', 16//, 'NDK_DEBUG=1'
+    }
+
+    task cleanNative(type: Exec) {
+        def ndkbuild
+        if (rootProject.hasProperty("ANDROID_NDK_HOME")) {
+            ndkbuild = rootProject.property("ANDROID_NDK_HOME")
+            ndkbuild += '/'
```
Comment (liaxim) [inline GVRf/Framework/framework/build.gradle]: '/' is supposed to work on Windows too

**Comment by thomasflynn** [comment]
can someone who regularly builds on windows please cherry-pick this and verify it?  thanks.

**Comment by deepakrawat22** [comment]
I tested on windows and it builds.

**Comment by thomasflynn** [comment]
thank you!

**Pull request merged**